helix-gateway/src/main/java/org/apache/helix/gateway/service/HelixGatewayServiceProcessor.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.helix.gateway.service;
+
+/**
+ * Translate from/to GRPC function call to Helix Gateway Service event.
+ */
+public interface HelixGatewayServiceProcessor {
+
+  public boolean sendStateTransitionMessage(String instanceName);
+
+  public void pushEventToManager(GatewayServiceManager.GateWayServiceEvent event);
```

Comment:
Let's be consistent with naming?

send or push.

Reply:
TFTR. Updated.